docs/Server/API/access_model.md

```diff
@@ -17,12 +17,35 @@ Any client, regardless of authentication, is able to READ a dataset with the
 `access` property set to `"public"`. Only the owner of the dataset, or a user with
 the administrator *role*, can UPDATE or DELETE the dataset.
 
+## User Identity
+
+The Pbench Server works with an external OIDC identity provider, for example
+Red Hat, Google, or GitHub SSO. The Pbench Server authenticates user access using
+the OIDC JWT tokens. The token's encoded `sub` field is the primary "user ID"
+controlling ownership of and access to data. The token's `preferred_username`
+field will be remembered and used when trying to report the "user ID" in human
+readable form.
```

Comment:
"remembered" strikes me funny...I would use "is cached" or "is stored".

Reply:
Why? Is it funny that I can't remember where I cached the store? 😛